ui/webui/src/components/AnacondaAboutModal.jsx

```diff
@@ -0,0 +1,102 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with This program; If not, see <http://www.gnu.org/licenses/>.
+ */
+import cockpit from "cockpit";
+
+import React, { useState, useEffect } from "react";
+import {
+    AboutModal,
+    Button,
+    Flex,
+    TextContent,
+    TextList,
+    TextListItem,
+    Stack,
+    StackItem
+} from "@patternfly/react-core";
+import { ExternalLinkAltIcon } from "@patternfly/react-icons";
+
+import { read_os_release as readOsRelease } from "os-release.js";
+import { getAnacondaVersion } from "../helpers/product.js";
+
+const _ = cockpit.gettext;
+
+const AboutModalVersions = ({ anacondaVersion }) => {
+    const arr = [["Anaconda", anacondaVersion]];
+
+    return (
+        <TextContent>
+            <TextList id="about-modal-content" component="dl">
+                {arr.map((item) => (
+                    <React.Fragment key={item[0]}>
+                        <TextListItem component="dt">{item[0]}</TextListItem>
+                        <TextListItem component="dd">{item[1]}</TextListItem>
+                    </React.Fragment>
+                ))}
+            </TextList>
+        </TextContent>
+    );
+};
+
+const ProductName = ({ prettyName }) => {
+    return (
+        <Stack hasGutter>
+            <StackItem>{prettyName}</StackItem>
+            <StackItem className="subtitle">{_("Powered by Anaconda")}</StackItem>
+        </Stack>
+    );
+};
+
+export const AnacondaAboutModal = ({ isModalOpen, setIsAboutModalOpen }) => {
+    const toggleModal = () => {
+        setIsAboutModalOpen(!isModalOpen);
+    };
+
+    const [anacondaVersion, setAnacondaVersion] = useState("");
+    const [prettyName, setPrettyName] = useState("");
+    useEffect(() => {
+        getAnacondaVersion().then(content => setAnacondaVersion(content));
+        readOsRelease().then(osRelease => setPrettyName(osRelease.PRETTY_NAME));
+    }, []);
+
+    return (
+        <>
+            <AboutModal
+              noAboutModalBoxContentContainer
+              isOpen={isModalOpen}
+              onClose={toggleModal}
+              trademark="Anaconda © 2023 Red Hat, Inc."
+              productName={<ProductName prettyName={prettyName} />}
+            >
+                <Flex id="about-modal-flex2" direction={{ default: "column" }} justifyContent={{ default: "justifyContentSpaceBetween" }}>
+                    <AboutModalVersions anacondaVersion={anacondaVersion} />
+                    <Flex id="about-modal-flex">
+                        <Button
+                          isInline
+                          id="anaconda-page-button"
+                          variant="link"
+                          icon={<ExternalLinkAltIcon />}
+                          href="https://github.com/rhinstaller/anaconda"
```

Comment:
@garrett @jkonecny12 what 're we expected to use as the project page?

Reply:
Yes, I would go with the project page. We can also show documentation but we have a link to that on the project page so I think that is better option.